sc/src/main/java/uk/ac/ebi/atlas/experimentpage/ExperimentPageContentService.java

```diff
@@ -66,6 +58,13 @@ public JsonObject getTsnePlotData(String experimentAccession) {
         return result;
     }
 
+    public JsonArray getTsnePlotMetaData(String experimentAccession) {
```

Comment:
What is this method actually doing? Why do you need to a create an JSON object, add something to it, then retrieve it immediately afterwards? What is the purpose of this intermediate step?

Reply:
Agreed. It was a useless intermediate step. Fixed the logic.

Reply:
Why do you need the logic to be in a private method anyway? That would only be useful if you are calling the `getMetadata` in multiple different places, which you are not doing.